alias

```diff
@@ -134,3 +134,13 @@ revoke-my-ip-all =
   !f() {
     aws revoke-my-ip ${1} all all
   }; f
+
+get-vpc-id-by-name =
+  !f() {
+      aws ec2 describe-vpcs --filter "Name=tag:Name,Values=${1}" --query 'Vpcs[*].{id:VpcId}' --output text
+  }; f
+
+get-vpc-name-by-id =
+  !f() {
+      aws ec2 describe-vpcs --filter "Name=vpc-id,Values=${1}" --query 'Vpcs[*].Tags[].Value' --output text
```

Comment:
This looks like we are including values of all tags. I think we should only be including the ``Name`` tag in what is returned.

Reply:
I would also recommend to include only the tag `Name`. +1 to include these 2 in the aliases.